templates/Makefile.erb

```diff
@@ -37,13 +37,13 @@ file-list: file-list-before-build <%= @name %>-project
 
 <%- if @bill_of_materials -%>
 <%= @bill_of_materials %>:
-	mkdir -p '<%= @bill_of_materials.path %>'
-	mv bill-of-materials '<%= @bill_of_materials.path %>'
+	mkdir -p "<%= @bill_of_materials.path %>"
```

Comment:
I think windows handles double quotes better then single quotes while dealing with spaces. This also could have been me desperately trying to get something to work and is unnecessary, but I'm not sure